.github/workflows/ci.yml

```diff
@@ -906,6 +898,11 @@ jobs:
         run: python -m pytest modin/pandas/test/test_io.py --verbose
         if: matrix.test-task == 'modin/pandas/test/test_io.py'
       - uses: codecov/codecov-action@v2
+      - name: Stop local ray cluster
```

Comment:
It's good practice to explicitly release resources.

Reply:
Same thing should be done in push.yml.